docs/editing.md

```diff
@@ -174,6 +184,20 @@ This doesn’t mean that you’re the editor, just that you’ve been suggested
 
 ![Editorial flow](images/JOSS-flowchart.png)
 
+## Expectations on JOSS editors
+
+### Responding to editorial assignments
+
+As documented above, papers will be assigned to you by one of the EiCs. We ask that EiCs respond in a timely fashion (< 72 hours) to invites to edit a new submission.
+
+### Continued attention to assigned submissions
+
+As an editor, part of your job is to ensure that submissions you're responsible for are progressing smoothly through the editorial process. This means that once or twice per week you should check your GitHub notifications and/or your editorial dashboard (e.g. `http://joss.theoj.org/dashboard/arfon`) for updates to the papers you are handling.
+
+**If reviews go stale**
+
+Sometimes reviews go quiet, either because a reviewer has failed to complete their review or an author has been slow to respond to a reviewer's feedback. **It's your job as the editor to prompt the author/or reviewer(s) to revisit the submission if there has been no response within 7-10 days.**
```

Comment:
unless there's a clear statement in the review thread that says an action is coming at a slightly later time, perhaps because a reviewer committed to a review by a certain date, or an author is making changes and says they will be done by a certain date.